Assets/Scripts/Manager/StorySceneManager.cs

```diff
@@ -371,6 +370,8 @@ public class StorySceneManager : MonoBehaviour {
 		while (textBoxManager.isActive) {
 			yield return null;
 		}
+
+        humanController.allowInput = true; //Set allow input to true so the player can exit the human doll
```

Comment:
Oh yes this is a fix to the final scene bug where you cannot switch out of the human for the good ending